operator/pkg/reconciliation/construct_statefulset.go

```diff
@@ -0,0 +1,162 @@
+// Copyright DataStax, Inc.
```

Comment:
Just like construct_service.go, there should be no logic changes in this file either.